.github/workflows/C++.yml

```diff
@@ -0,0 +1,79 @@
+name: C++ Build
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - v*
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  Linux:
+    runs-on: ubuntu-latest
+    container: quay.io/pypa/manylinux2014_x86_64:latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Install fftw
+      run: |
+        yum install -y fftw3-devel
+
+    - name: Compile C++ code
+      run: |
+        make lib
+        make spreadtestall
+        make test
+
+  MacOS_clang:
+    runs-on: macos-latest
+    env:
+      MACOSX_DEPLOYMENT_TARGET: 10.15
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Install omp and fftw
+      run: |
+        brew install libomp fftw
+        cp make.inc.macosx_clang make.inc
+
+    - name: Compile C++ code
+      run: |
+        make lib
+        make spreadtestall
+        make test
+
+  MacOS_gcc:
+    runs-on: macos-latest
+    env:
+      MACOSX_DEPLOYMENT_TARGET: 10.15
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Install gcc and fftw
+      run: |
+        brew install gcc@10 fftw
```

Comment:
Installation instructions say gcc-8. Should we use that instead?

Reply:
I guess it's up to Mac users to choose which version gcc compiler to use.  Maybe we should change the Installation instructions

Reply:
Right. We should make sure that our test procedure corresponds as closely as possible with the installation instructions, so either they should change or this should.